Description: Fix "Conflicting types of variable inverse_mouse to cause undefined behaviour"
Bug-Debian: http://bugs.debian.org/751510
Author: Fabian Greffrath <fabian+debian@greffrath.com>

--- a/rott/rt_menu.c
+++ b/rott/rt_menu.c
@@ -5359,9 +5359,9 @@ void DrawExtOptionsMenu (void)
 
    FlipMenuBuf();
 }
-extern long inverse_mouse;
+extern int inverse_mouse;
 extern boolean usemouselook;
-extern int iG_aimCross;
+extern boolean iG_aimCross;
 extern boolean usejump;
 extern boolean sdl_fullscreen;
 
